docs/concepts/vega-chain.md

```diff
@@ -4,76 +4,75 @@ title: Vega Chain
 hide_title: false
 ---
 
-Vega uses Tendermint as a consensus layer to form a blockchain. The rest of the information here informs on how that blockchain and its relevant components is comprised. [Read more about how Vega bridges to Ethereum](/docs/concepts/vega-chain/#bridges-used-for-staking). 
+Vega uses Tendermint as a consensus layer to form a blockchain. The rest of the information here informs on how that blockchain and its relevant components is comprised. 
+
+Read more: [How Vega bridges to Ethereum](/docs/concepts/vega-chain/#bridges-used-for-staking)
 
 ## Delegated proof of stake
-Vega runs on a delegated proof of stake blockchain. Participants--validators and token-holders--use their VEGA tokens to nominate the validator nodes that run the network. Non-validator participants assign the voting rights of their VEGA tokens to endorse a validator's trustworthiness. 
+Vega runs on a delegated proof of stake blockchain. Participants -- validators and token-holders -- use their VEGA tokens to nominate validator nodes that run the network. Non-validator participants assign the voting rights of their VEGA tokens to endorse a validator's trustworthiness. 
```

Comment:
suggestion (non-blocking) I wonder if this can do more to explain the slightly novel (to me) way that Vega does POS... that doesn't slash bad validator's stake but instead reduces the rewards they would have received from trading fees that epoch....
Token holders nominate validators they believe to be trust worthy, if a validator turns out to do things badly it (and it's delegates), forfeit staking rewards from trading fees that epoch. Then token holders may change their nomination, resulting in fewer rewards for un-trustworthy validators.

Reply:
Agree with above suggestion."Vega is non slashing, i.e., there is no mechanism through which a token holder can lose a delegated token through a validator being punished; any measures to that end use different mechanism that will hurt a bad validators (and potentially their delegators) revenue, but not the delegated tokens.

Reply:
Non-slashing added!